action.yml

```diff
@@ -129,6 +135,7 @@ runs:
         echo FC=$FC>>$GITHUB_ENV
         echo CC=$CC>>$GITHUB_ENV
         echo CXX=$CXX>>$GITHUB_ENV
+        echo "MKLLIB=$MKLLIB" >> $GITHUB_ENV
```

Comment:
if `MKLLIB` and `MKLROOT` are set by the oneAPI activation script, the `export_intel_vars` function in `setup-fortran.sh` should handle them, no need to set separately here.

However the previous step may need modification to support cache restore on Windows, `set | findstr /c:"oneAPI" >> "%GITHUB_ENV%"` may not cover everything we need